packages/build-utils/src/lambda.ts

```diff
@@ -24,6 +32,8 @@ export interface LambdaOptionsBase {
   supportsMultiPayloads?: boolean;
   supportsWrapper?: boolean;
   experimentalResponseStreaming?: boolean;
+  operationType?: OperationType;
```

Comment:
How would `operationType: 'API'` be different than `bundledFunction: { API: 1 }`? Is it to differentiate the fact that bundling happened at all or not?

Reply:
Do we have a doc discussing the goals for these fields? `bundledFunction: { API: 1 }` seems a bit vague/limiting unless I'm misunderstanding, it could provide additional context with a different shape/fields

Reply:
I think exposing bundled functions is out of scope for this change. I am going to open a separate discussion about if and how we'd want to handle this.

I've updated the PR to use only single values.